ClaudiaHernandezGarcia/30Aug/HackerRank/arrayList/Solution.java

```diff
@@ -0,0 +1,40 @@
+package arrayList;
+
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Solution {
+    public static void main(String args[]) {
+        Scanner scan = new Scanner(System.in);
+        int lines = scan.nextInt();
+        List<ArrayList<Integer>> queryArray = new ArrayList<ArrayList<Integer>>();
```

Comment:
I think you mean List<List<Integer>> Right ??

Reply:
I was wondering if both should have the interface. So should the list always be used, right?